goal_src/jak2/levels/city/common/civilian.gc

```diff
@@ -403,7 +403,12 @@
         (dotimes (s0-0 s2-0)
           (+! f30-0 (civilian-method-214 obj (-> s3-0 branch-array s0-0) (+ arg1 -1) arg2 s1-0))
           )
-        (set! arg3 (+ s1-0 (the float (/ (the int f30-0) s2-0))))
+        (set! arg3
```

Comment:
Is there a preferred format for indicating a manual patch?

Reply:
Not really, just a small comment saying something like e.g. `;; changed to fix civilian crash` is sufficient just so we know that this section was modified by hand in case we go back and modify the decompiled output of this file.